src/settings/BatchGroupConfigurationSettings/BatchVouchers/BatchVouchersContainer.js

```diff
@@ -0,0 +1,38 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+
+import { batchVouchersFromPropResource } from '../../../common/resources';
+import BatchVouchers from './BatchVouchers';
+
+const BatchVouchersContainer = ({ mutator, batchVoucherId }) => {
+  const downloadBatchVouchers = useCallback(
+    () => {
+      mutator.batchVouchers.GET();
+      // then download file
```

Comment:
but the api return json or xml, it wouldn't work with regular get

Reply:
added logic